src/main/java/stream/flarebot/flarebot/mod/ModlogEvent.java

```diff
@@ -0,0 +1,36 @@
+package stream.flarebot.flarebot.mod;
+
+public enum ModlogEvent {
+    MEMBER_JOIN,
```

Comment:
What we could and should do is have a getModlogEventEmbed. Then just change the enum name, set the colour to a param passed in here and any other things we need in all cases. Then return a EmbedBuilder for futt

Reply:
would that really be worth it at that point? I mean the only thing that would do is centralize the creation of the embed, but other then that all the embeds have different fields and prams.

Reply:
Would be better for modification, colour changes, keeping capitalisation consistency, spelling consistency etc

Reply:
Well the main use i see for this is adding things that are needed in all cases witch all we have their is the footer is the same. I mean yegh the embed title we could get from the embed. and if we had all of it be the same color that chould be useful but other then that I don't see the use. If you still think it's a good idea I'll go ahead and do it but i want you to think about where really this whould be usefull because I don't think it'd be

Reply:
It wouldn't be all the same colour which means this is even better, we pass a colour into the enum and just get it from that. So much can be reduced code wise to just have the base embed generated already.

Reply:
Ok